bridge-fungible-token/src/bridge_fungible_token_contract.sw.sw

```diff
@@ -0,0 +1,164 @@
+contract;
```

Comment:
Looks like a typo in the file name (`.sw.sw`). I'd also drop the `_contract` at the end since it feels redundant